examples/stable/mongodb/blueprint-v2/mongo-blueprint.yaml

```diff
@@ -16,7 +16,7 @@ actions:
       objects:
         mongosecret:
           kind: Secret
-          name: '{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-mongodb'
+          name: '{{ .StatefulSet.Name }}'
```

Comment:
This may not work if the `auth.existingSecret` option is used during installation.
```
# Install
$ helm3 install my-release bitnami/mongodb --namespace mongo --set architecture="replicaset" --set auth.existingSecret=mongodb

# Check secret
$ k get secrets -n mongo
NAME                               TYPE                                  DATA   AGE
default-token-mbdg7                kubernetes.io/service-account-token   3      2m34s
mongodb                            Opaque                                2      2m20s
my-release-mongodb-token-qhvwr     kubernetes.io/service-account-token   3      36s
sh.helm.release.v1.my-release.v1   helm.sh/release.v1                    1      36s

# Statefulset
$ root@83329f9485eb:/repo# k get statefulset -n mongo
NAME                         READY   AGE
my-release-mongodb           1/2     46s
my-release-mongodb-arbiter   1/1     46s
```